flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java

```diff
@@ -972,7 +972,7 @@ private static void setupNotificationChannel(
       notificationChannel.setGroup(notificationChannelDetails.groupId);
       if (notificationChannelDetails.playSound) {
         AudioAttributes audioAttributes =
-            new AudioAttributes.Builder().setUsage(AudioAttributes.USAGE_NOTIFICATION).build();
+            new AudioAttributes.Builder().setUsage(notificationChannelDetails.audioAttributesUsage).build();
```

Comment:
This is the place to coalesce a null value from a scheduled notification to `AudioAttributes.USAGE_NOTIFICATION`